lib/config/apiConfig.ts

```diff
@@ -0,0 +1,5 @@
+import { ApiStrategy } from '@lib/constants/apiStrategy';
+
+const raw = process.env.NEXT_PUBLIC_API_STRATEGY;
+
+export const API_STRATEGY: ApiStrategy = raw === 'grpc' ? ApiStrategy.GRPC : ApiStrategy.REST;
```

Comment:
Well, there's actually no point in dedicated variable "raw". May be simplified.
```suggestion
export const API_STRATEGY: ApiStrategy = process.env.NEXT_PUBLIC_API_STRATEGY === 'grpc' ? ApiStrategy.GRPC : ApiStrategy.REST;
```

Reply:
resolved